plugins/woocommerce-blocks/tests/e2e/tests/price-filter/price-filter.block_theme.side_effects.spec.ts

```diff
@@ -79,17 +105,17 @@ test.describe( `${ blockData.name } Block - with All products Block`, () => {
 		const maxPriceInput = page.getByRole( 'textbox', {
 			name: 'Filter products by maximum price',
 		} );
+		// await page.pause();
 
 		await maxPriceInput.dblclick();
-		await maxPriceInput.fill( '$10' );
+		await maxPriceInput.fill( '$5' );
```

Comment:
I added this change because the order of how E2E tests are executed is changed: [the Mini Cart E2E test](https://github.com/woocommerce/woocommerce/blob/acebcc07bdf721863d09e86029afacfdc8f4b34b/plugins/woocommerce-blocks/tests/e2e/tests/mini-cart/mini-cart-block.shopper.block_theme.side_effects.spec.ts/#L115-L120) does a side-effect that impacts this test as well (it changes the price).

I'm aware that this is a workaround, but it doesn't make sense to invest in a long-term solution because we will re-thing how we clean the test env after a side-effect in the next cooldown: https://github.com/woocommerce/woocommerce/issues/45347

cc @nielslange @WunderBart

Reply:
Sounds good to me. That's exactly what we've discussed in p1711023323470099/1711019541.954049-slack-C02UBB1EPEF.